src/database/services/keycloak-api-service.ts

```diff
@@ -1,12 +1,37 @@
-import type { User } from "../../types/keycloak/user";
 import fetch from "node-fetch";
 
+/**
+ * Interface for a KeycloakProfile. 
+ * 
+ * FIXME: This is from node_modules/keycloak-js/lib/keycloak.d.ts
+ *  This a temporary solution to avoid the error: "Cannot find module 'keycloak-js'"  
+ */
+
+export interface KeycloakProfile {
+	id?: string;
+	username?: string;
+	email?: string;
+	firstName?: string;
+	lastName?: string;
+	enabled?: boolean;
+	emailVerified?: boolean;
+	totp?: boolean;
+	createdTimestamp?: number;
+	attributes?: Record<string, unknown>;
+}
+/**
+ * Custom Interface for a user in keycloak functions with severaGuid added.
+ */
+
```

Comment:
There is no need for a line break between the Docs and the thing that is being documented. I keep seeing this